mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/CheckCmd.scala

```diff
@@ -72,8 +68,8 @@ class CheckCmd(name: String = "check", description: String = "Check a TLA+ speci
 
     logger.info("Tuning: " + tuning.toList.map { case (k, v) => s"$k=$v" }.mkString(":"))
 
-    executor.passOptions.set("general.tuning", tuning)
-    executor.passOptions.set("checker.nworkers", nworkers)
+    executor.passOptions.set("general.tuning", tuning) // XXX
+    executor.passOptions.set("checker.nworkers", nworkers) // XXX
```

Comment:
What's `XXX`? 🙃

Reply:
That marks these stringly typed settings for eradication :dagger: ;) -- but it should not be in this PR.